kubernetes/Config.cmake.in

```diff
@@ -0,0 +1,5 @@
+find_package(OpenSSL REQUIRED)
+find_package(ZLIB REQUIRED)
+find_package(yaml CONFIG REQUIRED)
+
+include("${CMAKE_CURRENT_LIST_DIR}/@pkgName@Targets.cmake")
```

Comment:
What is the purpose of the file "@pkgName@Targets.cmake" ?

Reply:
Here is how it is defined in [CMake's docs](https://cmake.org/cmake/help/git-stage/guide/importing-exporting/index.html#creating-a-package-configuration-file):

> The file contains code suitable for use by downstreams to import all targets listed in the install command from the installation tree.

@pkgName@ would be replaced by the value of the variable pkgName when configure_file() is invoked.